### PR TITLE
Entries: Added `normalizer` documentation

### PR DESCRIPTION
**Pull request by Arkni**
Ref: jzaefferer/jquery-validation#1609

### CODE REVIEW

File: entries/normalizer.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+<entry name="normalizer" type="method" return="String">
+	<title>normalizer</title>
+	<desc>Change/transform the value of the element before validating.</desc>
+	<longdesc>
+		The user can change the value of an element before validating the element in question. The new value will be then used by the associated methods instead of the real one.
+		<p>This method should return a String value, otherwise it will throw a TypeError exception.</p>
+		<p>Note that this method desn't change the element's value, it's only change the value passed to the plugin.</p>
```
Comment (staabm) [inline entries/normalizer.xml]: Desnt -> doesnt
Element's -> elements'
It's -> it
Change -> changes
Passed to the plugin -> used for validation

File: entries/normalizer.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+<entry name="normalizer" type="method" return="String">
+	<title>normalizer</title>
+	<desc>Change/transform the value of the element before validating.</desc>
+	<longdesc>
+		The user can change the value of an element before validating the element in question. The new value will be then used by the associated methods instead of the real one.
```
Comment (staabm) [inline entries/normalizer.xml]: Below you say it wont change the elements value :-)

File: entries/normalizer.xml (hunk 1 above)
Comment (Arkni) [inline entries/normalizer.xml]: Ops.

File: entries/normalizer.xml (hunk 2 above)
Comment (Arkni) [inline entries/normalizer.xml]: Make sense :)

Comment (Arkni) [inline entries/normalizer.xml]: Do you have a better desc for this ?

File: entries/normalizer.xml [hunk 3]
```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+<entry name="normalizer" type="method" return="String">
+	<title>normalizer</title>
+	<desc>Change/transform the value of the element before validating.</desc>
```
Comment (staabm) [inline entries/normalizer.xml]: I would use something like: "Prepares/transforms the elements value for validation."

File: entries/normalizer.xml (hunk 2 above)
Comment (staabm) [inline entries/normalizer.xml]: Also: other entries in the docs are more neutral and less prose.

Comment (staabm) [inline entries/normalizer.xml]: other entries also dont repeat in long-desc what was already mentioned in the desc.

I think we could just mention the restrictions (string type) and the arguments beeing passed to it (the value) and the adjusted "this" context

File: entries/normalizer.xml (hunk 3 above)
Comment (Arkni) [inline entries/normalizer.xml]: Simpler :+1:

File: entries/normalizer.xml (hunk 2 above)
Comment (Arkni) [inline entries/normalizer.xml]: That means I can omit the logdesc ?

Comment (Arkni) [inline entries/normalizer.xml]: > I think we could just mention the restrictions (string type) and the arguments beeing passed to it (the value) and the adjusted "this" context
ops, I forgot to mention the argument.

Comment (staabm) [inline entries/normalizer.xml]: Long-desc should contain
- normalizer doesnt change the elements value
- normalizer gets the value passed as argument
- normalizer needs to return a string
- "this" within a normalizer references the corresponding dom-element

Comment (Arkni) [inline entries/normalizer.xml]: I will copy pasta your comment :D (without the comment about the server side, as it's obvious)

**Comment by Arkni** [comment]
Done (with a little improvement) 0e1e573

**Comment by staabm** [comment]
@Arkni thx, perfect

**Comment by Arkni** [comment]
You're welcome :)

**Comment by staabm** [comment]
@Arkni FYI, just deployed the docs: http://jqueryvalidation.org/normalizer/

**Comment by Arkni** [comment]
Good to know, Thanks :)

> I guess we need to add some links from other pages so this new docs can be found more easily

Where we can link to this feature ? in methods ? `validate()`'s docs ?

**Pull request merged**